modules/tinymce/src/core/test/ts/browser/keyboard/EnterKeyTest.ts

```diff
@@ -15,6 +15,7 @@ describe('browser.tinymce.core.keyboard.EnterKey', () => {
     extended_valid_elements: 'div[id|style|contenteditable],span[id|style|contenteditable],#dt,#dd',
     entities: 'raw',
     indent: false,
+    text_patterns: false,
```

Comment:
Why were these disabled here? If tests were failing it seems like it might be legitimate so I don't think we should be disabling these here.

Reply:
I figured textpatterns changes enter behaviour, there are tests elsewhere to check that works, so these should cover the non-textpatterns behaviour only. Fair point that I should figure out why these were failing though, and see if something needs fixing. Or at least if these failed before, if you included textpattern plugin...

Reply:
It could even be an ordering thing, whereby moving it into core is causing issues which was my biggest concern.

Reply:
Checked, and this fails in the same way on develop if you add the TextPattern plugin to the config. Considering EnterKeyListsTest had the same error, I assume it's the same. Unsure if fixing whatever is causing that is suitable scope for this ticket, considering that.

Reply:
Logged a Jira to look into this, and added TODO comments in the two test files